components/published-toggle/index.jsx

```diff
@@ -1,10 +1,11 @@
-import React, { useState } from 'react'
+import React, { useState, useCallback } from 'react'
 import { classNames } from '@oacore/design/lib/utils'
 
 import styles from './index.css'
 
 const PublishedToggle = ({
   className,
+  onVisibilityChanged,
```

Comment:
It's a form element. Simple `onChange` and `checked` would be enough.

Reply:
It's designed like that for now. Internally it's just input element but outside it shouldn't look like that.